weather.js

```diff
@@ -0,0 +1,36 @@
+var city= "Beograd"
+
+function myFunction() {
```

Comment:
Please rename "myFunction" to something more meaningfull

Reply:
Renamed it to loadData